#!/usr/bin/env python
from dynamic_reconfigure.parameter_generator_catkin import *

PACKAGE = 'minrob_cv'

gen = ParameterGenerator()

gen.add('crop_video', bool_t, 0, "Crop video to indicated frame range", False)
gen.add('start_frame', int_t, 0, "Starting frame (zero-based)",     0, 0, 500)
gen.add('end_frame',   int_t, 0, "Ending frame (zero-based)",     500, 0, 500)
gen.add('fps',         int_t, 0, "Framerate in Hertz",             25, 1,  25)

exit(gen.generate(PACKAGE, "minrob_cv", "ImagePublisher"))
